crypto/helpers.go

```diff
@@ -0,0 +1,82 @@
+package crypto
+
+import (
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/pem"
+	"math/big"
+	"os"
+	"time"
+)
+
+func RandomCertificateSerialNumber() (*big.Int, error) {
```

Comment:
I think "random" should at least be in the name to indicate the source of the actual number. But I don't feel that strongly about it. I'll change it to Generate for now.